browser/ui/views/commands/default_accelerators_mac.mm

```diff
@@ -53,6 +53,13 @@ AcceleratorMapping ToAcceleratorMapping(NSMenuItem* item) {
   NSString* keyEquivalent = item.keyEquivalent;
   DVLOG(2) << __FUNCTION__ << item.tag << " > "
            << base::SysNSStringToUTF16(keyEquivalent);
+
+  unsigned short keyCode = [keyEquivalent characterAtIndex:0];
+  if ([keyEquivalent isEqualToString:@"\t"]) {
+    // characterAtIndex returns 'v' for \t. So we should set keyCode manually.
```

Comment:
ascii code of `\t` is `9`. and `9` is used as `v`'s keycode :)

Reply:
Nice catch 👍 was using char code not keycode.